.README/rules/sort-tags.md

```diff
@@ -8,24 +8,32 @@ Sorts tags by a specified sequence according to tag name.
 
 ##### `tagSequence`
 
-An array of tag names indicating the preferred sequence for sorting tags.
+An array of tag group objects indicating the preferred sequence for sorting tags.
+
+Each item in the array can be an object with a `tags` property set to an array
+of tag names.
 
 Tag names earlier in the list will be arranged first. The relative position of
 tags of the same name will not be changed.
 
+Earlier groups will also be arranged before later groups, but with the added
```

Comment:
"that additional line breaks may be added between (or before or after) such groups." -> "that one line break will be added between groups"?

It is just one line break, and it will be always be added in its current form, right?

Reply:
The default is 1 line, but it can be controlled with `linesBetween`. I realized the latter was missing from the docs, so just added that too.

I'm not sure what you mean "always be added in its current form". If you mean that the line break follows the asterisk and spacing of the surrounding context, then yes.

Reply:
Marking as resolved though feel free to comment further as needed.